src/app/pages/property-details/reviews/reviews.component.ts

```diff
@@ -0,0 +1,64 @@
+import { Subscription } from 'rxjs';
+import { Component, OnInit, Input, OnDestroy } from '@angular/core';
+import { ClientReviewsService } from 'src/app/services/client-reviews/client-reviews.service';
+import { Router } from '@angular/router';
+import { NgxSpinnerService } from 'ngx-spinner';
+import { environment } from 'src/environments/environment.prod';
```

Comment:
What do you think of importing from the `environment.ts` file and not the `.prod` file. I think it's a good practice to have angular make that switch for us in the build process  instead of doing it ourselves

Reply:
That was an error. Thank you for pointing it out